config/boards/orangepi3-lts.conf

```diff
@@ -0,0 +1,8 @@
+# Allwinner H6 quad core 2GB RAM SoC GBE USB3
+BOARD_NAME="OPI 3 LTS"
```

Comment:
I suggest to rename it to `Orange Pi 3 LTS` since the original board's name is "Orange Pi 3"

Reply:
OK I'll go with that. I'm all for tidiness! Haha!